corehq/apps/app_manager/models.py

```diff
@@ -415,10 +415,10 @@ class FormActions(DocumentSchema):
 
     def all_property_names(self):
         names = set()
-        names.update(self.update_case.update.keys())
-        names.update(self.case_preload.preload.values())
+        names.update(list(self.update_case.update.keys()))
```

Comment:
I think most of these can be changed to eliminate `.keys()` (and sometimes even `list(...)`), so this would become
```py
names.update(self.update_case.update)
```
In general `list(dict_instance)` is equivalent to `list(dict_instance.keys())`

Understood if this is not something you want to address now since these are automated changes.